### PR TITLE
Do not use 'eval' (fix #44)

### LINKED ISSUE #44: Eval is not the best practice and it may lead to errors
Hi!

I'm working on Electron application and it uses svg.js and svg.select.js libs. The Electron documentation advises to disable or override eval. So the app has content-security-policy metatag that disables eval. It lead to errors in svg.select.js lib, because it uses eval. 
Do you have any plans to change it? Thanks!

### PR DESCRIPTION
**Pull request by clusterberries**
Fix issue #44

### CODE REVIEW

**Comment by clusterberries** [comment]
@Fuzzyma please review. Thanks!

File: src/svg.select.js [hunk 1]
```diff
@@ -11,23 +11,30 @@ function SelectHandler(el) {
...
     }
 
     this.pointCoords = function (point, object) {
-      var settings = this.pointsList[point];
-
       return {
-        x: this.pointCoord(settings[0], object),
-        y: this.pointCoord(settings[1], object)
+        x: this.pointCoord(point, 0, object),
+        y: this.pointCoord(point, 1, object)
```
Comment (Fuzzyma) [inline src/svg.select.js]: I thought about a maybe easier way to do that :)

what about:

```js
var settings = this.pointsList[point];

return {
  x: this.pointCoord(settings[0], object, (point === 't' || point === 'b')),
  y: this.pointCoord(settings[1], object, (point === 'r' || point === 'l'))
}
```

And then in the pointCoord function you can simply check teh third argument. If its true, half the value :)

Comment (clusterberries) [reply src/svg.select.js]: @Fuzzyma sure, I'm ok with that. I already pushed the changes.
I updated commit using 'commit --amend'. I'm new in open source contributing, please let me know if I did something wrong :)

**Comment by Fuzzyma** [comment]
woah that loosk so much nicer, doesnt it? :)
Thank you very much! Ammending is totally ok for such small changed!

**Comment by clusterberries** [comment]
Hi @Fuzzyma ! Could you please create a new build with this fix? Thanks!

**Pull request merged**